templates/docker.tmpl

```diff
@@ -74,6 +74,18 @@
   basicAuth = [{{range getBasicAuth $container}}
     "{{.}}",
   {{end}}]
+  {{if getRequestHeaders $container}}
```

Comment:
Using the parse method for checking if request headers exist means we'll emit the log outputs twice: once in the check and once in the actual extraction (line 79). This might be somewhat confusing.

It might be better do introduce a little `hasRequestHeaders` template function, similar to how we do it with other functions.

Reply:
kk.